repository/src/main/resources/alfresco/model/contentModel.xml

```diff
@@ -1408,6 +1408,17 @@
             </property>
          </properties>
       </aspect>
+      <!-- This aspect marks a node with some or all renditions stored as properties -->

```

Comment:
Trivial: add comment to clearly mark this as "experimental - unsupported (not for production-use)" or similar.

Use same comment through-out, including the new repository property (a-g.p).